packages/papi/papi.0.1.1/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+maintainer:   ["David Kaloper Meršinjak <dk505@cam.ac.uk>"]
+authors:      ["David Kaloper Meršinjak <dk505@cam.ac.uk>"]
+license:      "ISC"
+homepage:     "https://github.com/pqwy/ocaml-papi"
+doc:          "https://pqwy.github.io/ocaml-papi/doc"
+dev-repo:     "git+https://github.com/pqwy/ocaml-papi.git"
+bug-reports:  "https://github.com/pqwy/ocaml-papi/issues"
+synopsis:     "Performance Application Programming Interface (PAPI) bindings"
+
+build: [ [ "dune" "subst" ] {pinned}
+         [ "dune" "build" "-p" name "-j" jobs ]
+         [ "dune" "runtest" ] {with-test} ]
+
+depends: [
+  "ocamlfind" {build}
+  "dune" {build & >= "1.7"}
+  "fmt" {with-test} ]
+
+depexts: [
+  ["libpapi-dev"] {os-distribution = "debian"}
+  ["libpapi-dev"] {os-distribution = "ubuntu"}
+  ["papi-devel"] {os-distribution = "centos"}
+  ["papi"] {os-distribution = "archlinux"}
```

Comment:
```suggestion
  ["papi"] {os-distribution = "arch"}
```
on Archlinux, the `os-distribution` variable is `arch` not `archlinux`

Reply:
Ok, thanks!